rackcity/views/change_plan_views.py

```diff
@@ -307,7 +307,11 @@ def change_plan_page_count(request):
     Return total number of pages according to page size, which must be
     specified as query parameter.
     """
-    return get_page_count_response(ChangePlan, request.query_params)
+    user = request.user
+    user_change_plans = ChangePlan.objects.filter(owner=user)
+    return get_page_count_response(
```

Comment:
page count was not being filtered by change plan owner

Reply:
i think we wrote the EXACT same code to fix this bug lmao
so probably no conflicts